node_modules/browser/page-switch.js

```diff
@@ -12,17 +12,21 @@ const {Deck} = require('./deck');
 const {select, remove} = require('./deck/actions');
 
 const readThumbnailURI = uri =>
-  'none' && `url(https://raw.githubusercontent.com/gordonbrander/website-tiles/master/${url.getDomainName(uri)}.png)`;
+  'none' && `https://raw.githubusercontent.com/gordonbrander/website-tiles/master/${url.getDomainName(uri)}.png`;
 
 const equals = x => y => x.equals(y)
 
 const Tab = Component(({items, item}) =>
-  DOM.div({
+  DOM.img({
     key: `tab-${item.get('id')}`,
     className: 'tab',
-    style: {
-      backgroundImage: readThumbnailURI(item.get('location'))
-    },
+    src: readThumbnailURI(item.get('location')),
+    onError: item.get('thumbnail') && (event => {
+      event.target.src = URL.createObjectURL(item.get('thumbnail'));
```

Comment:
Don't forget to revoke. And can't the URL be created earlier? Right after retrieving the blob.